liquibase-couchbase/src/test/java/system/change/InsertDocumentsFromFileSystemTest.java

```diff
@@ -146,10 +156,10 @@ private static String getDocId(JsonObject doc) {
     @Test
     @SneakyThrows
     void Should_generate_expression_key() {
-        Liquibase liquibase = liquibase(EXPRESSION_KEY_GENERATOR_TEST_XML);
+        Liquibase liquibase = liquibase(INSERT_EXPRESSION_KEY_GENERATOR_TEST_XML);
 
         Assertions.assertThatNoException().isThrownBy(liquibase::update);
-        assertThat(collection).extractingDocument("testKey::id1::0").hasField("extraField");
-        assertThat(collection).extractingDocument("testKey::id2::1").hasField("extraField");
+        assertThat(collection).extractingDocument("testKey::id1::0").jsonHasField("extraField");
```

Comment:
Could we extract duplicated field name to String constant? The same - about the keyspace of the document

Reply:
Done